### ISSUE TITLE
population field for per capita analysis

### ISSUE THREAD

**Issue by smaprs** [CSSEGISandData/COVID-19#975]
Amazing work, thank you all!
Just a suggestion: would it be nice to add a population field for each country (when it is known) to make for per capita analysis?
Thank you again and again

**Comment by peterdrier** [comment]
Good idea, and there are plenty of sources out there for population data.  And some sites are already combining the two.  Perhaps given the workload these kind folks are under it's best to do that join yourself.

**Comment by emdeex** [comment]
I'd like to integrate this as well.  Can you ref some good sites that have the population data in a suitable format?

**Comment by ivanMSC** [comment]
Wikipedia

**Comment by peterdrier** [comment]
https://lmgtfy.com/?q=population+by+country

**Comment by emdeex** [comment]
There is this https://github.com/thewiremonkey/factbook.csv
It's 5 years old, and doesn't breakdown Countries by State/Provinces.  

Individual country census data has State/Province breakdowns, but still haven't found a single global compendium.  If I make any progress, I'll post it here.

**Comment by smaprs** [comment]
Yes, I've taken all countries population from Wikipedia
https://en.wikipedia.org/wiki/List_of_countries_and_dependencies_by_population
Then sorten according CSSEGISandData list, and added respective columns
Even if not 100% accurate, it's enough
